src/main/java/hudson/plugins/scm/koji/model/RPM.java

```diff
@@ -86,6 +92,29 @@ public boolean hasUrl() {
         return url != null;
     }
 
+    public void setHashSum(File file) throws IOException, NoSuchAlgorithmException {
+        if (file == null) {
+            hashSum = null;
+            return;
+        }
+        byte[] buffer = new byte[(int) file.length()];
+        FileInputStream stream = new FileInputStream(file);
+        stream.read(buffer);
+        stream.close();
+        MessageDigest messageDigest = MessageDigest.getInstance("MD5");
+        messageDigest.update(buffer);
+        byte[] digest = messageDigest.digest(buffer);
+        StringBuilder builder = new StringBuilder();
+        for (byte bytes : digest) {
```

Comment:
you can convert `byte[]` to `String` as one-liner with `DatatypeConverter#printHexBinary(byte[])`